### PR TITLE
Added slack invite link.

### PR DESCRIPTION
**Pull request by mooreds**
Removed the writing link as we're pretty good for now.

TODO before merging:

* set up a slack bot to welcome folks and send them the code of conduct: https://github.com/fharper/warm-welcome/
* update the image with one from David.

### CODE REVIEW

File: site/community.html [hunk 1]
```diff
@@ -39,6 +39,18 @@ <h2>Ask Questions</h2>
           <strong>Forums</strong>&nbsp;&nbsp;&nbsp;&nbsp;<i class="orange-text fas fa-long-arrow-right"></i>
         </footer>
       </a>
+      <a href="https://join.slack.com/t/fusionauth/shared_invite/zt-6ykfov9u-MTwA1A6ptv0Gr6KcFBVG4w" class="col-xs-12 col-lg-3 gray gray-split-card">
+        <header>
+          <img src="/assets/img/stamps/forums.svg" alt="Ask Questions">
+          <h2>Join Slack</h2>
+        </header>
+        <div class="content">
+          <p>Join the FusionAuth Slack community to meet and chat with other FusionAuth users</p>
```
Comment (robotdan) [inline site/community.html:48]: The other 3 tiles on this page end their description with a period. Was this one intentional?

File: site/community.html [hunk 2]
```diff
@@ -39,6 +39,18 @@ <h2>Ask Questions</h2>
           <strong>Forums</strong>&nbsp;&nbsp;&nbsp;&nbsp;<i class="orange-text fas fa-long-arrow-right"></i>
         </footer>
       </a>
+      <a href="https://join.slack.com/t/fusionauth/shared_invite/zt-6ykfov9u-MTwA1A6ptv0Gr6KcFBVG4w" class="col-xs-12 col-lg-3 gray gray-split-card">
+        <header>
+          <img src="/assets/img/stamps/forums.svg" alt="Ask Questions">
```
Comment (robotdan) [inline site/community.html:44]: We can keep this svg or see if David has any others that we could swap in here?

**Review by robotdan — COMMENTED** [review]
Looks good. Do we want to have a disclaimer anywhere to better describe the amount of FusionAuth interaction to expect?

File: site/community.html (hunk 1 above)
Comment (mooreds) [reply site/community.html:48]: Good catch.

File: site/community.html (hunk 2 above)
Comment (mooreds) [reply site/community.html:44]: I pinged David, we'll want a new SVG.

**Comment by mooreds** [comment]
I am planning to install a slackbot to welcome new users to the slack. That slackbot will do two things:

* let them know about the code of conduct (https://fusionauth.io/community/forum/topic/1000/code-of-conduct)
* let them know what level of interaction to expect from FusionAuth employees (basically, none). 

Here's provisional language:

```
Thanks for joining the FusionAuth slack. This is a venue for FusionAuth community members to meet, hang out and collaborate. Welcome!

Two things to keep in mind:

* Please abide by the FusionAuth community code of conduct: https://fusionauth.io/community/forum/topic/1000/code-of-conduct . If you can't do so, please don't join this community.
* While FusionAuth employees will be on this slack, they won't be answering support questions. If you need support, please post in the community forum: https://fusionauth.io/community/forum/ or open a support ticket: https://account.fusionauth.io/account/support/ (if you have a paid plan which includes support).

Thanks!
- The FusionAuth team
```

**Comment by jobannon** [comment]
I think that we should add a monthly reminder to the channel as well.  Unless the slack bot will display the message in the general channel and not send the message to the individual.  If that is the case, then we likely do not need a monthly reminder.

Also, Grammarly has two edits/suggestions you could consider for the provisional message.

```
This is your monthly reminder -- This is a venue for FusionAuth community members to meet, hang out, and collaborate. Welcome!.  Two things to keep in mind:

* Please abide by the FusionAuth community code of conduct: https://fusionauth.io/community/forum/topic/1000/code-of-conduct. If you can't do so, please don't join this community.
* While FusionAuth employees will be on this slack, they won't be answering support questions. If you need support, please post in the community forum: https://fusionauth.io/community/forum/ or open a support ticket: https://account.fusionauth.io/account/support/ (if you have a paid plan which includes support).

Thanks!
- The FusionAuth team
```

**Comment by mooreds** [comment]
Thanks @jobannon . I would like to wait on the reminder until we see folks asking for support. Heck, if folks want to ask for support, that's perfectly fine, we just won't be giving it to them in the slack.

I'm trying to strike the balance between "welcome to the community" and "we won't be using this for support, sorry" withouth erring too far on either side.

**Comment by mooreds** [comment]
But if folks need automated reminding, will look into a way to send this message regularly.

**Pull request merged**